jerry-core/ecma/builtin-objects/ecma-builtin-string-prototype.c

```diff
@@ -1388,6 +1426,16 @@ ecma_builtin_string_prototype_dispatch_routine (uint16_t builtin_routine_id, /**
       ret_value = ecma_builtin_string_prototype_object_iterator (to_string_val);
       break;
     }
+    case ECMA_STRING_PROTOTYPE_PAD_END:
```

Comment:
Combine `ECMA_STRING_PROTOTYPE_PAD_END` with `ECMA_STRING_PROTOTYPE_PAD_START` and call directly `ecma_string_pad`.